.github/workflows/ADD _ISSUES_TO_PROJECT

```diff
@@ -13,6 +13,6 @@ jobs:
       - uses: actions/add-to-project@RELEASE_VERSION
         with:
           project-url: https://github.com/orgs/saving-satoshi/projects/2
-          github-token: ${{ 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 }}
+          github-token: ${{ secrets.SS_ISSUES2TASK }}
           labeled: 
           label-operator: 
```

Comment:
@ecurrencyhodler I did notice one more thing, these fields (labeled, label-operator) need to be populated 
Take a look at [this example](https://github.com/actions/add-to-project#example-usage-issue-opened-with-labels-bug-or-needs-triage)

Reply:
Thanks. I was wondering if that might break things. 

But I'm slightly skeptical because it wasn't a required field listed in their https://github.com/actions/add-to-project/blob/main/action.yml

It also would mean issues without labels wouldn't work.

I posted a question in the forum.  https://github.com/orgs/community/discussions/45389